src/query/executor/transform/lazy.go

```diff
@@ -145,6 +145,23 @@ type lazyBlock struct {
 	processedBlock block.Block
 }
 
+// Unconsolidated returns the unconlidated version for the block
```

Comment:
`s/unconlidated/unconsolidated`

Reply:
`s/for the block/of the block`

Reply:
we use "for the block" in most other places.